File: 1_two_sum/Solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if((nums[i] + nums[j]) == target):
                    return [i, j]
        
# obj = Solution()
# lst = [1,2,3,4,5]
# target = 7
# l = obj.twoSum(lst, target)
# print(l)

File: 1_two_sum/Solution.cpp
// #include <iostream>
// #include <vector>
// using namespace std;

class Solution{
public:
    vector<int> twoSum(vector<int> &nums, int target){
        for(int i = 0; i < nums.size(); i++){
            for(int j = i + 1; j < nums.size(); j++){
                if(nums.at(i) + nums.at(j) == target){
                    return vector<int>{i, j};
                }
            }
        }
        return vector<int>();
    }
};

// int main()
// {
//     vector <int> vctr{1,2,3,4,5}, result;
//     Solution obj;
//     result = obj.twoSum(vctr, 9);
//     for (auto i = result.begin(); i != result.end(); i++) {
//         std::cout << *i << " ";
//     }
//     return 0;
// }
